rom/laghos.cpp

```diff
@@ -359,6 +359,11 @@ int main(int argc, char *argv[])
     const bool usingWindows = (numWindows > 0 || windowNumSamples > 0);
     if (usingWindows)
     {
+        if (romOptions.dimX  > 0) romOptions.max_dimX  = romOptions.dimX;
+        if (romOptions.dimV  > 0) romOptions.max_dimV  = romOptions.dimV;
+        if (romOptions.dimE  > 0) romOptions.max_dimE  = romOptions.dimE;
+        if (romOptions.dimFv > 0) romOptions.max_dimFv = romOptions.dimFv;
+        if (romOptions.dimFe > 0) romOptions.max_dimFe = romOptions.dimFe;
```

Comment:
If I understand correctly, this means that in the window case, in which we usually do not specify these dimensions, now the dimensions mean maximum dimensions. It would be helpful to add a comment or put this in the description in the corresponding `args.AddOption` calls.

Reply:
Yes. Usually, in the online phase, in the serial ROM we specify the reduced dimension, and in the time windowing case we let the reduced dimensions be defined from reading the time window parameter file. Now, in this branch, if we specify the reduced dimensions in the time windowing case, it is used to control the actual ROM dimension in each window by setting the variables `romOptions.max_dim*`, and the actual dimension `romOptions.dim*` is taken as the smaller one among `romOptions.max_dim*` and the value read from the time parameter file.